### PR TITLE
 West Midland 3-Nonye Ehiguese- Git and GitHub-Lesson 2

### PR DESCRIPTION
**Pull request by GemPearl**
Implemented feedback on Question 5 and 15.

# Your Details

- Your Name: Nonye Ehiguese
- Your City: West Midland
- Your Slack Name: GemPearl

# Homework Details

- Module: Git and GitHub
- Week: 2

-----
[View rendered homework.md](https://github.com/GemPearl/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Comment by Haseeb3927** [comment]
Great effort Nonye 👍  keep it up.

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,112 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Cascading Style Sheets
```
Comment (uzma000) [inline homework.md]: The question is what the CSS useful for not stand for. I think you misunderstood the question :)

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,112 @@
...
 
-When you send a file over the internet
+Pull request is communicate to other the changes you have pushed to a branch on a repository on GitHub.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+History log is a feature of Git that shore each commits . Thus provide a chain of commits.
```
Comment (uzma000) [inline homework.md:80]: great answer and also it allows us to see the history about the changes we made.

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,112 @@
...
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+To Push means to update or publish the remote branch with local commit.
```
Comment (uzma000) [inline homework.md:86]: I believe you meant, sending the changes you made in your local repository to the remote one.

**Review by uzma000 — COMMENTED** [review]
All answers were great and good effort

File: homework.md (hunk 2 above)
Comment (MireilleNyoubissi) [reply homework.md:80]: Same difference, yes.

File: homework.md [hunk 4]
```diff
@@ -4,110 +4,113 @@
...
+Help share information
+Enable People to make changes and share back 
+make Checkpoint and save work as we go along
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Forking a repository means you are taking the repository into your GitHub account
```
Comment (MireilleNyoubissi) [inline homework.md:63]: I think it is important to mention that "to fork" a repository means "to make a copy" of that repository from another Github account to yours. The point here is to be able to make changes to your copy without affecting the original repository.

File: homework.md [hunk 5]
```diff
@@ -4,110 +4,113 @@
...
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+When you clone a repository the file from your GitHub account is downloaded and saved your local physical storage
```
Comment (MireilleNyoubissi) [inline homework.md:69]: I believe you can clone a repository from any account, not just yours. Yes, the point here is to download a copy of that repository and save it in your local storage. However, it is important to note that the downloaded copy will retain a link to its original repository therefore if you attempt to implement your own changes, you may face protection measures.

File: homework.md [hunk 6]
```diff
@@ -4,110 +4,113 @@
...
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull request is communicate to other the changes you have pushed to a branch on a repository on GitHub.
```
Comment (MireilleNyoubissi) [inline homework.md:75]: Or, in other words, a method of submitting contributions to an open development project.

File: homework.md [hunk 7]
```diff
@@ -4,110 +4,113 @@
...
+JavaScript is a just-in-time complied Programming language well known as thescripting language for web pages and many non-browser enviromental such as Node.js, Apache CouchDB and Adobe Acrobat.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Help share information
+Enable People to make changes and share back 
+make Checkpoint and save work as we go along
```
Comment (MireilleNyoubissi) [inline homework.md:57]: I think version control is too important to be omitted

**Review by MireilleNyoubissi — COMMENTED** [review]
You did well! I learn a lot by reviewing your work and, I hope that our contributions would also ease your understanding of Git/Github. It is a fantastic journey.

**Pull request closed without merging**